engine/src/main/java/org/terasology/world/sun/BasicCelestialModel.java

```diff
@@ -29,9 +29,21 @@
     private static final long DUSK_TIME = 3 * DAY_LENGTH / 4;
     private static final long MIDNIGHT_TIME = 0;
 
+    /**
+     * Return the direction of the sun based on the time of the day.<br />
+     * At days=0 midnight the direction of the sun will be -90 degrees, while at
+     * days=0.5 noon it will be +90 degrees.
+     * @param days the time of day
+     * @return the direction of the sun in radians in the range [-PI;PI]
+     */
     @Override
     public float getSunPosAngle(float days) {
```

Comment:
Sorry, we need to rename this method to getMainLightElevation as elevation is what it returns. Also, we need to rewrite the javadoc in terms of both sun and moon as the MainLight implements both. Talking only in terms of sun would confuse a reader unfamiliar with the simplified celestial model Terasology uses.